src/configParser.ts

```diff
@@ -24,13 +24,20 @@ export type Config = {
   url: string;
 };
 
+const fallbackConfig: Config = {
+  assignments: {},
+  cookieDomain: '',
+  cookieName: '',
+  experienceSamplingWeight: 0,
+  splits: {},
+  url: ''
+};
+
 class ConfigParser {
   getConfig(): Config {
-    if (typeof window.atob === 'function') {
-      return JSON.parse(window.atob(window.TT));
-    } else {
-      return JSON.parse(base64.decode(window.TT));
-    }
+    const decodedConfig = atob(window.TT);
+    if (decodedConfig) return JSON.parse(decodedConfig);
+    return fallbackConfig;
```

Comment:
so previously would an invalid base64 string have raised an exception?

i think there's an advantage to that in the direct feedback of a misconfiguration or something. since testtrack tries to provide a good experience in the face of network issues, it might be hard to detect that something was not set up properly if we just return an empty config.

Reply:
> so previously would an invalid base64 string have raised an exception?

Yes. If we want to raise and let the library consumer handle the exception, that would replicate the current expectations.